internal/weldr/api.go

```diff
@@ -2393,7 +2402,7 @@ func (api *API) resolveOSTreeCommits(sourceSpecs map[string][]ostree.SourceSpec,
 					Checksum: checksum,
 				}
 			} else {
-				commit, err := ostree.Resolve(source)
+				commit, err := ostree.Resolve(source, resolveConfiguration())
```

Comment:
I am not sure where to load configuration for this resolve call, help me to solve this I am using ENV variables for the time being but that is not correct I guess this is some API process?